weblate/templates/snippets/meta-css.html

```diff
@@ -22,8 +21,7 @@
 {% endif %}
 {% endcompress %}
 {% if fonts_cdn_url %}
-  <link rel="stylesheet" href="{{ fonts_cdn_url }}font-source/source-sans-3.css" />
-  <link rel="stylesheet" href="{{ fonts_cdn_url }}font-source/source-code-pro.css" />
+  <link rel="stylesheet" href="{{ fonts_cdn_url }}fonts/fonts.css" />
 {% endif %}
```

Comment:
This won't work with existing CDN, but I think it's time to ditch this option completely

Reply:
Alright